src/main/java/org/opentripplanner/standalone/config/buildconfig/DemConfig.java

```diff
@@ -29,31 +44,45 @@ located outside the local filesystem (including cloud storage services) or at va
           the command line.
           """
         )
-        .asObjects(DemConfig::mapDemExtract)
+        .asObjects(nodeAdapter -> mapDemExtract(nodeAdapter, defaults))
     );
   }
 
-  private static DemExtractParameters mapDemExtract(NodeAdapter config) {
-    return new DemExtractParametersBuilder()
+  private static DemExtractParameters mapDemExtract(
+    NodeAdapter config,
+    DemExtractParameters defaults
+  ) {
+    String documentationAddition = " Overrides the value specified in `demDefaults`.";
+    return mapGenericDemParameters(config, defaults, documentationAddition)
       .withSource(
         config.of("source").since(V2_2).summary("The unique URI pointing to the data file.").asUri()
       )
+      .build();
+  }
+
+  private static DemExtractParametersBuilder mapGenericDemParameters(
+    NodeAdapter node,
+    DemExtractParameters defaults,
+    String documentationAddition
+  ) {
+    return defaults
+      .copyOf()
       .withElevationUnitMultiplier(
-        config
+        node
           .of("elevationUnitMultiplier")
-          .since(V2_2)
-          .summary("Specify a multiplier to convert elevation units from source to meters.")
+          .since(V2_3)
```

Comment:
Should this be V2_2 or V2_3 or different for defaults/specific dem files?

Reply:
I think the parameter has been around for a long time. Not sure if the version means the current place or parameter in general. You could argue for both.